packages/react/src/react-component-lib/utils/attachProps.ts

```diff
@@ -62,17 +62,6 @@ export const getClassName = (classList: DOMTokenList, newProps: any, oldProps: a
   return finalClassNames.join(' ');
 };
 
-/**
- * Transforms a React event name to a browser event name.
- */
-export const transformReactEventName = (eventNameSuffix: string) => {
-  switch (eventNameSuffix) {
-    case 'doubleclick':
-      return 'dblclick';
-  }
-  return eventNameSuffix;
-};
-
```

Comment:
Think this needs to stay in for stencil v4. If you install and build again it should add it back